CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+# How to contribute
+
+This module was developed in to support Puppet training. It is open to
+the public so that students can see the code that goes in to creating
+the classroom experience. Although most of the development is done by 
+Puppet employees we welcome outside contributions. Be aware that your
+suggested change may not fit in with our future plans, so even very good
+code may not be accepted.
+
+Priority will be given to contributions from Puppet Professional Services
+engineers and Service Delivery Partners actively teaching classes.
+
+There are a few guidelines that we need contributors to follow so that 
+we can have a chance of keeping on top of things.
+
+## Getting Started
+
+* Make sure you have a [Jira account](https://tickets.puppetlabs.com)
+* Make sure you have a [GitHub account](https://github.com/signup/free)
+* Submit a ticket for your issue, assuming one does not already exist.
+  * Clearly describe the issue including steps to reproduce when it is a bug.
+  * Make sure you fill in the earliest version that you know has the issue.
+* Fork the repository on GitHub
+
+## Making Changes
+
+Releases are made from the `release` branch, which should always be in a
+releasable state. `master` is the active development branch and changes 
```

Comment:
agreed. I'm a little hesitant to be *too* prescriptive right off the bat. I'm ok with leaving some things a little vague so we have a little flexibility to find out what works.